src/together/cli/api/finetune.py

```diff
@@ -93,6 +93,12 @@ def fine_tuning(ctx: click.Context) -> None:
     default=False,
     help="Whether to skip the launch confirmation message",
 )
+@click.option(
+    "--train-on-inputs",
+    type=BOOL_WITH_AUTO,
+    default="auto",
+    help="Whether to mask the user messages in conversational data or prompts in instruction data",
```

Comment:
Nit: might be good to explain what happens in the default case and how `auto` is handled

Reply:
Up (you can just put `"auto" will automatically determine whether to mask the inputs based on the data format.` here)